app/Resources/translations/messages.de.xliff

```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="en" datatype="plaintext" original="file.ext">
```

Comment:
`target-language="en"` -> `target-language="de"` (I always forget to update this value too)